src/steps/malops/converters.ts

```diff
@@ -1,11 +1,32 @@
 import { createIntegrationEntity } from '@jupiterone/integration-sdk-core';
-import { Malop } from '../../types';
+import { Malop, SimpleMalopFeature } from '../../types';
 import { Entities } from '../constants';
 
 function getMalopKey(id: string): string {
   return `malop:${id}`;
 }
 
+// TODO: some improvements:
+// 1) If the value is boolean in string form - e.g. "false" or "true", convert it to boolean
```

Comment:
Tried the following fix. I was able to convert all of them to booleans but it causes a J1 Graph/schema error since we have defined this properties as of type `string` and not `boolean`. We could easily change their types in the types file, but wouldn't that cause the fetchMalops step to throw an error? (because they would be of type `string` originally)

```javascript
function toBool(val: any) {
  if (val === 'true') {
    return true;
  }

  if (val === 'false') {
    return false;
  }

  return val;
}

function mapMalopSimpleFieldValue(
  value: SimpleMalopFeature,
): boolean | string | string[] | null {
  if (!value) {
    return null;
  }
  // We know the return type isn't going to be an array
  if (value.totalValues && value.totalValues === 1) {
    if (value.values && value.values.length > 0) {
      return toBool(value.values[0]);
    }
  }

  return value.values ? value.values.map((val) => toBool(val)) : null;
}
```